cliquet/__init__.py

```diff
@@ -61,6 +62,8 @@
     'cliquet.storage_url': '',
     'cliquet.storage_max_fetch_size': 10000,
     'cliquet.userid_hmac_secret': '',
+    'cliquet.sentry_dsn': None,
```

Comment:
Since we have ``storage_url`` and ``cache_url``, maybe we should be use ``sentry_url`` too ? ... I know that storage_dsn and cache_dsn would have been better... :(
We'll also have ``statsd_<dsn|url>`` :)